src/views/Impressum.jsx

```diff
@@ -80,8 +80,8 @@ export default function Impressum() {
             <br />
           </p>
           <p>
-            Diese Website benutzt Google Analytics, einen Webanalysedienst der Google Inc. (''Google'').
-            Google Analytics verwendet sog. ''Cookies'', Textdateien, die auf Ihrem Computer gespeichert werden und die eine Analyse der Benutzung der
+            Diese Website benutzt Google Analytics, einen Webanalysedienst der Google Inc. (&apos;&apos;Google&apos;&apos;).
```

Comment:
I suggest we use double quotes?
```suggestion
            Diese Website benutzt Google Analytics, einen Webanalysedienst der Google Inc. (&quot;Google&quot;).
```